src/cobalt_model/site.rs

```diff
@@ -73,6 +77,11 @@ impl Site {
                 liquid::model::Value::scalar(kstring::KString::from_ref(base_url)),
             );
         }
+        attributes.insert(
+            "time".into(),
+            liquid::model::Value::scalar(kstring::KString::from_ref(&self.time.to_string())),
```

Comment:
There a reason you are inserting it as a string rather than doing `liquid::model::Value::scalar(self.time.clone())`?

Reply:
There isn't :raised_hands: 
Fixed.